modules/cnf-managing-device-interrupt-processing-for-guaranteed-pod-isolated-cpus.adoc

```diff
@@ -17,7 +17,7 @@ When configured, CRI-O disables device interrupts only as long as the pod is run
 [id="configuring-global-device-interrupts-handling-for-isolated-cpus_{context}"]
 == Configuring global device interrupts handling for isolated CPUs in Performance Addon Operator
 
-To configure Performance Addon Operator to disable global device interrupts, set the `GloballyDisableIrqLoadBalancing` field in the performance profile to *true*. When set to *true*, device interrupts are disabled on all isolated CPUs. When set to *true*, pods should not be annotated. A PerformanceProfile snippet illustrates this setting:
+To configure Performance Addon Operator to disable global device interrupts for the isolated CPU set, set the `GloballyDisableIrqLoadBalancing` field in the performance profile to `"true"`. A PerformanceProfile snippet illustrates this setting:
```

Comment:
performance profile snippet
We just recently updated our API object style guidlines: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-formatting